hugegraph-server/hugegraph-core/src/main/java/org/apache/hugegraph/backend/cache/CachedSchemaTransactionV2.java

```diff
@@ -85,8 +85,6 @@ public void close() {
     }
 
     private Cache<Id, Object> cache(String prefix, long capacity) {
-        // TODO: uncomment later - graph space
```

Comment:
maybe better to keep the **comment** due to we will import the graph-space feat later?